lib/src/lib/page-header/components/page-header/page-header.component.ts

```diff
@@ -21,13 +21,20 @@ export class Lab900PageHeaderComponent {
 
   @Input()
   public actions: ActionButton[];
+  public leftActions: ActionButton[];
+  public rightActions: ActionButton[];
 
   @Input()
   public data?: any;
 
   @Input()
   public breadCrumbs: BreadCrumb[];
 
+  public ngOnInit(): void {
+    this.leftActions = this.actions.filter((action) => action.align === 'left');
+    this.rightActions = this.actions.filter((action) => action.align === 'right' || action.align == null);
+  }
```

Comment:
Better to use ngOnChanges. Just in case the buttons change in the same lifecycle.

Reply:
updated